### DISCUSSION TITLE
Blackmagic Stills .dng import

### DISCUSSION THREAD

**Discussion by clodcd** [EatTheFuture/image_tools#14] [Questions & Troubleshooting] [answered]
Hi,
what's the correct way to import the black stills to detect the noise levels from a Blackmagic 6k?
Since the lut maker doesn't directly read .dng files how do I transform my stills? "Jpg Rec 709" or I have to use the BMPCC Film Gen 5 as Color Space Tag (which Davinci Resolve doesn't seem to have, idkw).
Thanks in advance!

**Comment by cessen** [comment, marked as answer]
The easiest thing to do is to convert your footage (including your dark images) to RGB first, rather than working with raw files.  I would recommend converting to whatever RGB space the camera manufacturer recommends for that camera.  In your case, I think(?) that would be Blackmagic's "Wide Gamut Gen 5" (actually the same as Gen 4) for the chromaticities and "Film Gen 5" for the transfer function.  But you should double-check.

If you can export the (RGB) dark images to either 16-bit tiff or 16-bit png, you should then be able to load them into LUT Maker.

Note: it's important that the settings you use for raw->RGB conversion (e.g. white balance) are the same between the footage and the dark images.  Otherwise the noise level estimates won't match, which may end up tinting or otherwise messing up your darks.  The simplest way to ensure that the settings are the same is to just convert all your footage from raw to RGB with exactly the same settings.  And then if you want to do further white balancing, etc., you can do that in RGB after you've already compensated for sensor noise.  It's not completely ideal, but in practice works just fine.

(The "ideal" workflow is to do the noise compensation in raw before converting to RGB, but we don't support raw in ETF Image Tools yet.  I can think of some ways to work around that, but it's probably not worth it, because the benefit over just doing everything in RGB is pretty small.)

**Reply by clodcd** [reply]
Thank you so much!
